chromium_src/chrome/common/media/cdm_registration.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/common/media/cdm_registration.h"
+
+#include "third_party/widevine/cdm/buildflags.h"
+
+#if BUILDFLAG_INTERNAL_BUNDLE_WIDEVINE_CDM() == 1
+#define BUNDLE_WIDEVINE_WAS_SET
+#endif
```

Comment:
We disabled `bundle_widevine_cdm` by default. `WidevineBuildFlag.FlagTest` test case checks it.
I think we don't need this file?

Reply:
I understand about `bundle_widevine_cdm`. But does it really mean that this whole file `cdm_registration.cc` is not needed? I believe we still need to replace `BUILDFLAG_INTERNAL_BUNDLE_WIDEVINE_CDM` by `BUILDFLAG_INTERNAL_ENABLE_WIDEVINE_CDM_COMPONENT`. As a related note, this comes from [snap-from-source 56c42943](https://git.launchpad.net/~chromium-team/chromium-browser/+git/snap-from-source/tree/build/chromium-patches/widevine-enable-version-string.patch?id=56c42943).

Reply:
As we already use widevine cdm component on linux, I thought we could use widevine cdm component's directory for our pre-installed location instead of using additional bundle define.
According to the below code, I assume that pre-installed widevine binary would work if it's stored in component directory?
```
#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
  bundled_widevine = GetBundledWidevine();
#endif

  content::CdmInfo* updated_widevine = nullptr;
#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
  updated_widevine = GetComponentUpdatedWidevine();
#endif
```

Reply:
With that, I think we could use component updater for arm64 w/o more work in the future.

Reply:
> According to the below code, I assume that pre-installed widevine binary would work if it's stored in component directory?

It is complicated to set up, but you are right. I will use this point to significantly simplify the PR.